src/Avatar/Avatar.js

```diff
@@ -77,6 +94,10 @@ Avatar.propTypes = {
    */
   alt: PropTypes.string,
   /**
+   * Used to render icon and text elements inside the Avatar.
+   */
+  children: PropTypes.node,
```

Comment:
Not keeping both, I just didn't want to barge in and change the API without discussing it first, so it left both options in place until everyone was agreed. I'll strip it back to just `children`.